ionic_user_interface/src/components/gym-selector/AutoComplete.vue

```diff
@@ -1,43 +1,60 @@
 <template>
-  <ion-input
-    v-bind="$attrs"
-    class="ion-text-end"
-    v-model="userInput"
-    @keyup.enter="onEnter"
-    @keydown.tab="onEnter"
-  />
-  <ion-card class="global-margin-left" v-if="filteredSuggestions.length && showCard">
-    <ion-text
-      class="suggestion-item"
+  <ion-item>
+    <ion-label class="global-absolute-position">{{ label }}</ion-label>
+    <ion-input
+      v-bind="$attrs"
```

Comment:
i think can remove this...? I added it cos vue was complaining about how i must add this if its not single root element

Reply:
done, and fixed some other console warnings as well